app.js

```diff
@@ -1,5 +1,37 @@
-// import functions and grab DOM elements
+/* eslint-disable quotes */
+let y = Math.floor(Math.random() * 20 + 1); 
+   
+// counting the number of guesses 
+// made for correct Guess 
+let number = 1; 
+
+document.getElementById("submit-guess").onclick = function(){ 
+  
+  
+    let x = document.getElementById("guessField").value; 
+
+    if (x === y) 
+    {     
+        alert("Congrats "
```

Comment:
Testing your app, it does seem that I'm unable to get the winning number(y) that you have here for the alert, it will only show the toolow or toohigh message even when I guess the number that is generated. Additional functionality will be needed here, per the rubric, there is a compareNumbers Function that you need to create and test, and think that will be helpful to start working this additional logic.